### PR TITLE
Fix `examples/status-message` readme

### PR DESCRIPTION
**Pull request by zavodil**

### CODE REVIEW

File: examples/status-message/README.md [hunk 1]
```diff
@@ -34,22 +34,27 @@ near js deploy --accountId status-message.test.near --base64File build/status-me
 </p>
 </details>
 
-
+## Set constants
+```sh
+export ACCOUNT_ID=alice..test.near
```
Comment (ailisp) [inline examples/status-message/README.md:39]:
```suggestion
export ACCOUNT_ID=alice.test.near
```

**Review by ailisp — APPROVED** [review]
Looks good. @volovyk-s may need to rebase to #100 after this?

File: jsvm/jsvm.c [hunk 2]
```diff
@@ -594,7 +594,7 @@ static void deduct_cost(uint64_t *cost) {
   uint64_t deposit[2];
   remaining_deposit(deposit, GET);
   if (u128_less_than(deposit, cost)) {
-    panic_str("insufficient deposit for storage");
+    // panic_str("insufficient deposit for storage");
```
Comment (volovyk-s) [inline jsvm/jsvm.c:597]: Probably we do not want to merge this.

File: examples/tic-tac-toe/jsconfig copy.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "experimentalDecorators": true,
+    },
+    "exclude": [
```
Comment (volovyk-s) [inline examples/tic-tac-toe/jsconfig copy.json:5]: Looks like a redundant file.

**Comment by volovyk-s** [comment]
@zavodil I have merged my examples refactoring. No boilerplate now, you need to add contract file, tests, and README, and that is it.

**Comment by volovyk-s** [comment]
@BenKurrek
This PR is outdated, but I like the `tik-tak-toe` contract. Should we add it to our examples here or as a separate repo?

**Comment by gagdiez** [comment]
I would recommend to add it here, at least until we figure out our "examples repo" situation: we have too many examples, and need to curate that repo.

**Pull request closed without merging**